File: CSharp/001_Program/Program.cs
// File Reader - Read and display file contents
using System;
using System.IO;

class Program
{
    static void Main()
    {
        Console.WriteLine("=== File Reader ===");
        Console.Write("Enter file path: ");
        string path = Console.ReadLine();
        
        try
        {
            string content = File.ReadAllText(path);
            string[] lines = File.ReadAllLines(path);
            
            Console.WriteLine("\n--- File Contents ---");
            Console.WriteLine(content);
            Console.WriteLine($"\n--- Statistics ---");
            Console.WriteLine($"Total lines: {lines.Length}");
            Console.WriteLine($"Total characters: {content.Length}");
            Console.WriteLine($"File size: {new FileInfo(path).Length} bytes");
        }
        catch (Exception ex)
        {
            Console.WriteLine($"Error: {ex.Message}");
        }
    }
}

File: Python/001_Program/program.py
#!/usr/bin/env python3
# File Reader - Read and display file contents
import os

def main():
    print("=== File Reader ===")
    filepath = input("Enter file path: ")
    
    try:
        with open(filepath, 'r') as f:
            content = f.read()
            lines = content.split('\n')
            
        print(f"\n--- File Contents ---")
        print(content)
        print(f"\n--- Statistics ---")
        print(f"Total lines: {len(lines)}")
        print(f"Total characters: {len(content)}")
        print(f"File size: {os.path.getsize(filepath)} bytes")
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    main()
